src/components/filter/__stories__/Filter.stories.tsx

```diff
@@ -16,11 +17,11 @@ import {
 import { FilterMultiSelect } from 'src/components/filter/filter-multi-select/FilterMultiSelect';
 import { FilterSelect } from 'src/components/filter/filter-select/FilterSelect';
 import { FilterText } from 'src/components/filter/filter-text/FilterText';
-import { useCountryOptions } from 'src/components/select/__stories__/useCountryOptions';
 import { type Flag, FlagIcon } from 'src/icons/flag-icon/FlagIcon';
 import { RemoveIcon } from 'src/icons/RemoveIcon';
-import type { CountryOption } from 'src/types/Country';
 import type { SelectOption } from 'src/types/SelectOption';
+import type { CountryOption } from 'src/utils/hooks/useCountryOptions';
+import { useCountryOptions } from 'src/utils/hooks/useCountryOptions';
```

Comment:
I'm surprised it didn't merge the type into one 🤔 Is this the new eslint rule? Disregard if it's actually is 😄 
```suggestion
import { type CountryOption, useCountryOptions } from 'src/utils/hooks/useCountryOptions';
```